.travis.yml

```diff
@@ -1,15 +1,18 @@
+dist: xenial
 language: node_js
 node_js:
-  - "6"
-  - "8"
-  - "10"
-sudo: false
-cache:
-  directories:
-    - node_modules
+- '10'
+- '8'
+cache: yarn
+services:
+- mysql
+# Don't run builds for renovate PRs
+if: NOT head_branch =~ ^renovate
```

Comment:
Renovate is not enabled on this repository yet, but should be soon so leaving this bit here